README.md

```diff
@@ -169,17 +153,10 @@ To avoid the complicated syntax all ISBN values or all values can be extracted a
 
 ## Matching and comparing numbers
 
-Correct number matching must result in 3.4 = 3.400 = 0.34e1 and correct number comparison must result in 10 > 2
+Correct number matching must result in 3.4 == 3.400 == 0.34e1 and correct number comparison must result in 10 > 2
 (in contrast to string comparison where "10" < "2").
 
-If the search term is a number then a numeric mode is used for "==", "<>", "<", "<=", ">", ">=", and "=".
-
-If the schema (see section [Schema](#schema) above) specifies a JSON field type
-then numeric mode is only used when that type is either `integer` or `number`.
-
-If the comparator is "=" and the JSON field type is not specified in the schema
-then numeric mode is only used if the actual JSONB type of the stored value is `number`
-(JSONB has no `integer` type).
+If the search term is a number then a numeric mode is used for "==", "<>", "<", "<=", ">",and ">=".
```

Comment:
Add "=". See compareNumber in CQL2PgJSONTest.java where both "address.zip= 4" and "address.zip==4" are a numeric comparison and and match "4.0" and "4e0".

Reply:
Add this sentence:

    If the comparator is "=" or "=="
    then numeric mode is only used if the actual JSONB type of the stored value is `number`
    (JSONB has no `integer` type).

See compareNumber and numberInStringField in CQL2PgJSONTest.java to see that it works that way.

Reply:
Not sure about that. Please see method in https://github.com/folio-org/cql2pgjson-java/blob/master/cql2pgjson/src/main/java/org/z3950/zing/cql/cql2pgjson/CQL2PgJSON.java#L901. It specifically excluded = for numeric mode. Also after eliminating legacy individual json schema, CQL to SQL translation for = and == will translate like this:
```
CQL: address.zip= 4 sortBy name
SQL: lower(f_unaccent(users.user_data->'address'->>'zip')) ~ lower(f_unaccent('(^|[[:punct:]]|[[:space:]]|(?=[[:punct:]]|[[:space:]]))4($|[[:punct:]]|[[:space:]]|(?<=[[:punct:]]|[[:space:]]))')) ORDER BY lower(f_unaccent(users.user_data->>'name'))
CQL: address.zip==4 sortBy name
SQL: ((jsonb_typeof(users.user_data->'address'->'zip')='number' AND users.user_data->'address'->'zip'=to_jsonb(4)) OR (jsonb_typeof(users.user_data->'address'->'zip')<>'number' AND users.user_data->'address'->'zip'='"4"' AND lower(f_unaccent(users.user_data->'address'->>'zip'))='4')) ORDER BY lower(f_unaccent(users.user_data->>'name'))
```

Reply:
Yes, you are right about "=". Only "==" is used for number comparison.

There is duplicated code in isNumberComparator and getNumberMatch that differs in this detail.

Reply:
The correct sentence should be

    If the search term is a number then a numeric mode is used for "==", "<>", "<", "<=", ">", and ">="
    if the actual JSONB type of the stored value is `number`
    (JSONB has no `integer` type).